vignettes/baselinenowcast.Rmd

```diff
@@ -270,40 +259,28 @@ ggplot(
   scale_y_reverse()
 ```
 
+Here, the grey indicates matrix elements that are `NA`, which we
+would expect to be the case in the bottom right portion of the reporting
+triangle where the counts have yet to be observed.
+
 # Estimate delay
 
 Now that we have the reporting triangle, we are now ready to pass it in to the
 `baselinenowcast` package to estimate the delay distribution.
 We will tell the function the maximum delay and the number of observations we
-want to use, though the default will be to use the whole reporting triangle.
-If the reporting triangle is too small for the user-specified delays and number
-of training observations, the function will throw an error.
+want to use for estimation.
 We only want to pass in the reporting triangle (for a single group!) to this
 function.
-If reference date are repeated because the reporting triangle contains multiple
-strata, the function will throw an error.
-
-The `get_delay_estimate()` function expects the following inputs:
-
-  - `reporting_triangle`: a matrix with the reporting triangle for a single
-  strata. Here the rows represent the time points and the columns represent the
-  observed delays, starting at 0. This can also be a reporting matrix or an
-  incomplete reporting matrix (so not all elements of the bottom right triangle
-  need to be missing).
-  - `max_delay`: an integer indicating the maximum delay to estimate. This must
-  be less than or equal to the number of rows in `triangle` minus 1, since we
-  assume `triangle` is indexed at 0.
-  - `n`: an integer indicating the number of observations by reference
-  date to use to fit the delay distribution. This must be less than or equal to
-  the number of rows in `triangle`. Here we will use `n_history_delay`.
-
+See documentation for `get_delay_estimate()` for a full description of the
```

Comment:
why here do we say see the docs and rely on a link and in the other cases we say look at ?